tests/testthat/test-conventional-commits.R

```diff
@@ -0,0 +1,7 @@
+test_that("Can parse conventional commits", {
+  repo <- withr::local_tempdir()
+  withr::local_dir(repo)
+  create_cc_repo(repo)
+  messages <- get_top_level_commits_impl(since = NULL)$message
+  expect_snapshot(collect_news(messages))
```

Comment:
For pretty snapshots:

```suggestion
  expect_snapshot(writeLines(collect_news(messages)))
```